os-eap70-openshift/added/standalone-openshift.xml

```diff
@@ -70,7 +70,7 @@
             </logger>
         </audit-log>
         <management-interfaces>
-            <http-interface http-upgrade-enabled="true" console-enabled="false"><!-- ##MGMT_IFACE_REALM## -->
+            <http-interface http-upgrade-enabled="true" console-enabled="false" ##MGMT_IFACE_REALM##>
```

Comment:
Is there a better way to do this?  We have other issues popping up about our version of standalone.xml not being valid xml.